packages/runtime/src/widgets/Form/Dropdown.tsx

```diff
@@ -61,12 +61,19 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
   const [selectedValue, setSelectedValue] = useState<
     string | number | undefined
   >();
+  const [dropdownStat, setDropdownStat] = useState<boolean>(false);
```

Comment:
Typo

Reply:
@evshi if you are talking here about `dropdownStat` then it is not typo, I set it as `stat` not `state`